2-Pricing/app.css

```diff
@@ -0,0 +1,355 @@
+:root {
+    --spacing: 1.875rem;
+    --featured-color: #EA580C;
+    --bravo-color: #EAB308;
+    --charlie-color: #22C55E;
+    --extra-color: #EF4444;
+}
+
+body {
+    background: no-repeat url("./images/Italian-streets.png");
+    background-size: cover;
+    background-position: center;
+    font-family: 'Manrope', sans-serif;
+    color: white;
+}
+
+#cart {
+    background: none;
+    border: none;
+    position: fixed;
+    top: 2rem;
+    right: 0.1rem;
+    transition: 0.2s ease-in-out;
+    z-index: 10;
+}
+
+#cart:hover { transform: scale(1.2); }
+
+main {
+    min-width: 18.75rem;
+    min-height: 100vh;
```

Comment:
`100vh` can cause problems in mobile devices because it doesn't account for the height of the navbar. We should use `height: 100%`.

Reply:
Great! I didn't know that using `vh` was problematic on mobile. Using `height: 100%` doesn't look good on large devices though (the image doesn't occupy the whole screen and the bottom of the screen is just a white rectangle). Maybe we could we apply `min-height: 100vh` for large devices?

Reply:
What about `min-height: 100%`? I think you should be able to achieve the same behavior using just 100% instead of 100vh. Maybe the container of this div is not extending the full height in large devices.

I would try to play a bit with the Chrome inspector, changing the CSS to see if we can make it work. We can try in the daily.

Reply:
Thanks! `height: 100%` did work in the end, I was just applying it in the wrong selector (had to apply first on the html tag, on the body tag). To summarize: it wasn't working because `main` is inside body (and body inside html) and the height was being cropped "earlier". Well spotted!